src/probability/kernel/disintegration.lean

```diff
@@ -0,0 +1,407 @@
+/-
+Copyright (c) 2023 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
+import probability.kernel.cond_cdf
+import measure_theory.constructions.polish
+
+/-!
+# Disintegration of measures on product spaces
+
+Let `ρ` be a finite measure on `α × Ω`, where `Ω` is a standard Borel space. In mathlib terms, `Ω`
+verifies `[nonempty Ω] [topological_space Ω] [polish_space Ω] [measurable_space Ω] [borel_space Ω]`.
+
+For any measurable space `γ`, there exists a kernel `cond_kernel ρ : kernel α Ω` such that we
```

Comment:
```suggestion
There exists a kernel `cond_kernel ρ : kernel α Ω` such that for any measurable space `γ`,
```
Also, this formulation feels a little bit strange to me: can you explain it without mentioning `γ`, i.e., saying that the rho-measure of any set is the integral with respect to rho.fst of the measure of the set along the fibers?